scripts/macos-portable.sh

```diff
@@ -16,6 +16,10 @@ DIST_DIR="$3"
 mkdir -p "$DIST_DIR"
 
 cp "$1" "$DIST_DIR/$LOCAL_CODA_EXE"
+codesign -s "$APPLE_ID" "$DIST_DIR/$LOCAL_CODA_EXE"
+cp "$2" "$DIST_DIR/$LOCAL_KADEMLIA"
```

Comment:
what is `LOCAL_KADEMLIA`, now that we're using libp2p?

Reply:
Good catch It's supposed to be LOCAL_LIBP2P i'll break this out into another PR

Reply:
OK, since there's already an approval with write access, I won't review further.